src/coreclr/tools/Common/Internal/Runtime/ReadyToRunConstants.cs

```diff
@@ -41,7 +41,7 @@ public enum ReadyToRunImportSectionFlags : ushort
     /// Constants for method and field encoding
     /// </summary>
     [Flags]
-    public enum ReadyToRunMethodSigFlags : byte
+    public enum ReadyToRunMethodSigFlags : ushort
     {
```

Comment:
This part is still problematic, I would not do this change in this PR so that we can merge something. Native AOT progress is blocked on this merging.